acceptance/internal/config.go

```diff
@@ -85,8 +86,10 @@ type ServerStub struct {
 	// The response body to return.
 	Response testserver.Response
 
-	// Artificial delay in seconds to simulate slow responses.
-	DelaySeconds *float64
+	// Artificial delay to simulate slow responses.
+	// Configure as "1ms", "2s", "3m", etc.
+	// See [time.ParseDuration] for details.
+	Delay time.Duration
```

Comment:
This should still be a pointer, otherwise you cannot override it from child directory with 0.

Reply:
You can put 0 and it won't sleep then
> A negative or zero duration causes Sleep to return immediately.
https://pkg.go.dev/time#Sleep

Reply:
Yes, I'm aware, I'm talking about merge algorithm. See https://github.com/databricks/cli/pull/2369 for context.

Reply:
The `ServerStub` is a list entry, so there is no way to override it.

Also, note that the pattern and response fields are not pointers.

Reply:
Right, I missed the part where it's part of ServerStub, I thought it's a global setting.